clients/admin-ui/src/features/common/form/inputs.tsx

```diff
@@ -440,6 +463,18 @@ const CreatableSelectInput = ({
           width: 3,
           height: 3,
         }),
+        singleValue: singleValueBlock
```

Comment:
this file has become quite messy, but this is basically copying over what was already in `SelectInput` into `CreatableSelectInput`